apps/web/pages/my-account/my-orders.vue

```diff
@@ -1,89 +1,133 @@
+          -->
+        </li>
+        <UiDivider class="col-span-3 -mx-4 !w-auto md:mx-0" />
+      </ul>
+      <table class="hidden md:block text-left typography-text-sm mx-4">
+        <caption class="hidden">
+          {{
+            $t('account.ordersAndReturns.listOfOrders')
+          }}
+        </caption>
+        <thead class="border-b-2 border-neutral-200">
+          <tr>
+            <th class="lg:py-4 py-2 lg:pr-4 pr-2 font-medium">{{ $t('account.ordersAndReturns.orderId') }}</th>
+            <th class="lg:p-4 p-2 font-medium lg:whitespace-nowrap">{{ $t('account.ordersAndReturns.orderDate') }}</th>
+            <th class="lg:p-4 p-2 font-medium">{{ $t('account.ordersAndReturns.amount') }}</th>
+            <th class="lg:p-4 p-2 font-medium">{{ $t('account.ordersAndReturns.shippingDate') }}</th>
+            <th class="lg:p-4 p-2 font-medium">{{ $t('account.ordersAndReturns.status') }}</th>
+            <!-- <th class="lg:py-4 py-2 lg:pl-4 pl-2"></th> -->
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="(order, index) in data.data.entries" :key="index" class="border-b border-neutral-200">
+            <td class="lg:py-4 py-2 lg:pr-4 pr-2 lg:whitespace-nowrap">{{ orderGetters.getId(order) }}</td>
+            <td class="lg:p-4 p-2 lg:whitespace-nowrap">{{ orderGetters.getDate(order) }}</td>
+            <td class="lg:p-4 p-2">{{ $n(orderGetters.getPrice(order), 'currency') }}</td>
+            <td class="lg:p-4 p-2">{{ orderGetters.getShippingDate(order) ?? '' }}</td>
+            <td class="lg:p-4 p-2 lg:whitespace-nowrap w-full">{{ orderGetters.getStatus(order) }}</td>
+            <!--
+            <td class="py-1.5 lg:pl-4 pl-2 text-right w-full">
+              <SfButton :tag="NuxtLink" size="sm" variant="tertiary" :to="`${paths.accountMyOrders}/${index}`">
+                {{ $t('account.ordersAndReturns.details') }}
+              </SfButton>
+            </td>
+            -->
+          </tr>
+        </tbody>
+      </table>
+
+      <UiPagination
+        :current-page="data.data.page"
+        :total-items="data.data.totalsCount"
+        :page-size="data.data.itemsPerPage"
+        :max-visible-pages="maxVisiblePages"
+      />
+    </div>
+  </NuxtLazyHydrate>
   <NuxtPage />
 </template>
 
 <script setup lang="ts">
+import { orderGetters } from '@plentymarkets/shop-sdk';
 import { SfButton } from '@storefront-ui/vue';
+import { useMediaQuery } from '@vueuse/core';
 
 definePageMeta({
   layout: 'account',
 });
+
+const isWideScreen = useMediaQuery(mediaQueries.desktop);
+const maxVisiblePages = ref(1);
+
+const setMaxVisiblePages = (isWide: boolean) => (maxVisiblePages.value = isWide ? 5 : 1);
+const route = useRoute();
+
+watch(isWideScreen, (value) => setMaxVisiblePages(value));
+onMounted(() => setMaxVisiblePages(isWideScreen.value));
+
 const { fetchCustomerOrders, data } = useCustomerOrders();
-await fetchCustomerOrders();
+// const NuxtLink = resolveComponent('NuxtLink');
```

Comment:
whats up with that?

Reply:
Is needed for the "Details" implementation

Reply:
ok 👍